doc/user_guide/user_guide.md

```diff
@@ -322,30 +393,32 @@ SELECT TE_SEQUENCE_CLASSIFICATION_SINGLE_TEXT_UDF(
     text_data
 )
 ```
-- Parameters:
-  - ```device_id```: To run on GPU, specify the valid cuda device ID. Otherwise,
-  you can provide NULL for this parameter.
-  - ```bucketfs_conn```: The BucketFS connection name
-  - ```sub_dir```: The directory where the model is stored in the BucketFS.
-  - ```model_name```: The name of the model to use for prediction. You can find the
-  details of the models in [huggingface models page](https://huggingface.co/models).
-  - ```text_data```: The input text to be classified
-
-The inference results are presented with predicted _LABEL_ and confidence
- _SCORE_ columns, combined with the inputs used when calling
-this UDF. In case of any error during model loading or prediction, these new
-columns are set to `null` and column _ERROR_MESSAGE_ is set
-to the stacktrace of the error. For example:
+
+* [Common Parameters](#common-udf-parameters):
+  * `device_id`
+  * `bucketfs_conn`
+  * `sub_dir`
+  * `model_name`
+* Specific parameters:
+  * `text_data`: The input text to be classified
+
+The output presents the input columns passed to the UDF plus additional columns containing the inference results, i.e. the predicted _LABEL_ and its confidence _SCORE_.
```

Comment:
the rest of the udfs still have the old wording regarding the results. do you want to update those to, or leave them as is? both is fine for me

Reply:
As we are on it now anyway, I will try to update the docs of the other UDF's, too.